deploy/examples/ingress-with-annotations.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
```

Comment:
We typically use the example name in this field: `name: ingress-with-annotations` would be more appropriate.

Still on the topic of naming: I think the example name (and file name) should actually be: `ingress-with-host`.